index.js

```diff
@@ -8,16 +8,19 @@ var debug = require('debug')('detective-sass');
  * @param  {String} fileContent
  * @return {String[]}
  */
-module.exports = function detective(fileContent) {
+module.exports = function detective(fileContent, opts) {
   if (typeof fileContent === 'undefined') { throw new Error('content not given'); }
   if (typeof fileContent !== 'string') { throw new Error('content is not a string'); }
+  if (Object.prototype.toString.call(opts) !== '[object Object]') { opts = {}; }
 
   var dependencies = [];
   var ast;
 
   try {
     debug('content: ' + fileContent);
-    ast = sass.parse(fileContent);
+    ast = sass.parse(fileContent, {
+      syntax: opts.syntax || 'sass'
```

Comment:
I don't consider that breaking. From #9, it sounds like this is more of a fix (though it does change the, broken, existing behavior).

Defaulting to sass is fine.